lib/rucio/db/sqla/models.py

```diff
@@ -1690,141 +1690,25 @@ class FollowEvents(BASE, ModelBase):
                    Index('DIDS_FOLLOWED_EVENTS_ACC_IDX', 'account'))
 
 
-def register_models(engine):
+def register_models(engine: Engine) -> None:
     """
-    Creates database tables for all models with the given engine
+    Creates database tables for all models in the `BASE.metadata` schema with the given engine.
+    Functionally this creates all models, as they all inherit from BASE and exist in the same `BASE.metadata` schema.
+
+    :param engine: engine to register the models with
+    :type engine: sqlalchemy.engine.Engine
```

Comment:
We haven't had a formal discussion on docstrings yet (context: https://github.com/rucio/rucio/issues/6628) but I think it's best if we avoid adding `type` information in the docstrings, as long as there's already type annotations in the function signature (like in this case), in order to avoid having to write duplicate information. 

Don't worry about it for this PR though; once a docstring guide is finalized and published then we can worry about standardising docstrings across the codespace, but for now this is a perfectly good docstring